packages/scripts/index.js

```diff
@@ -44,6 +45,14 @@ const startHandler = async ({ file }) => {
     // Rollup will be emitting lifecycle events for the bundling process such as start, end and error. These events are
     // used to control a spinner in the terminal for each `input-output` bundle.
     switch (event.code) {
+      case 'END':
+        if (push) {
+          console.log();
+          console.log(chalk.gray.bold('yalc push'));
+          const output = child_process.execSync('yalc push');
```

Comment:
Does `yalc` build the package first before pushing?

Reply:
`yalc push` does not. It uses the existing build, publishes and then pushes it. Since this would be called right after rollup finishes the build, `push` seemed the better option here.